lang/en.json

```diff
@@ -432,7 +432,7 @@
         "YearsDays": "%1$s years %2$s days",
         "Yes": "Yes",
         "YouAreCurrentlyUsing": "You are currently using Piwik %s.",
-        "YouAreViewingDemoShortMessage": "You are viewing the demo of Piwik",
+        "YouAreViewingDemoShortMessage": "You are viewing the demo of %1$sPiwik%2$s",
```

Comment:
It does not look like the translation key was modified?

Reply:
What do you mean?

Reply:
you wrote "I also updated the translation key YouAreViewingDemoShortMessage" but only  the string was updated. IIRC you need to change the translation key as well to ensure the string will be re-translated? (although in this case it may work because the translations will miss the `%1$s` and won't be valid)

Reply:
That's what I was wondering whether I have to change it or not. Ideally we would keep the name as it is descriptive but can simply change it.